up-transport-vsomeip/src/message_conversions.rs

```diff
@@ -540,11 +513,13 @@ impl VsomeipMessageToUMessage {
             UP_CLIENT_VSOMEIP_FN_TAG_CONVERT_VSOMEIP_MSG_TO_UMSG,
             request_id
         );
-        let req_id = rpc_correlation_registry.remove_ue_request_correlation(request_id)?;
+        let (req_id, sink) = rpc_correlation_registry.remove_ue_request_correlation(request_id)?;
```

Comment:
Sink is only used for traces?

Reply:
It's used a few lines below in order to build response `UMessage` using the `UMessageBuilder`.

Reply:
touche' my bad for code review on a phone